package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "hello-microverse",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "index.js",
+  "devDependencies": {
+    "hint": "^7.1.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/abiola-arobieke/Hello-Microverse.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/abiola-arobieke/Hello-Microverse/issues"
+  },
+  "homepage": "https://github.com/abiola-arobieke/Hello-Microverse#readme"
+}
```

Comment:
Kindly update the name of your feature branch to represent the general action that you performed in the branch e.g `setup-hello-project` 🙏. To do this, checkout to a new branch and make sure to give it a descriptive name. Then open a new pull request and ensure that you add a link to this pull request in the description 👍